product_supplierinfo_for_customer/i18n/de.po

```diff
@@ -0,0 +1,128 @@
+# Translation of Odoo Server.
```

Comment:
With the new weblate server you are not supposed to add .po files unless this is the first time.
Are you sure?

Reply:
Transifex added this one, do we have to delete it?

Reply:
I am not sure: @sbidoul ?

Reply:
Since it's a migration, it also the first time the .po file appear on this branch. So you can keep them, there is no risk of conflict.